addons/l10n_dk_nemhandel/models/account_move.py

```diff
@@ -0,0 +1,69 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    nemhandel_message_uuid = fields.Char(string='Nemhandel message ID')
+    nemhandel_move_state = fields.Selection(
+        selection=[
+            ('ready', 'Ready to send'),
+            ('to_send', 'Queued'),
+            ('skipped', 'Skipped'),
+            ('processing', 'Pending Reception'),
+            ('done', 'Done'),
+            ('error', 'Error'),
+        ],
+        compute='_compute_nemhandel_move_state', store=True,
+        string='Nemhandel status',
+        copy=False,
+    )
+
+    def _need_oioubl_21_xml(self):
+        self.ensure_one()
+
+        res = super()._need_ubl_cii_xml()
```

Comment:
```suggestion
        res = self._need_ubl_cii_xml()
```
I guess ? :thinking: